packages/pages/src/generate/features/features.ts

```diff
@@ -1,21 +1,15 @@
 import { ProjectStructure } from "../../common/src/project/structure.js";
 import { getTemplateFilepaths } from "../../common/src/template/internal/getTemplateFilepaths.js";
-import { loadTemplateModules } from "../../common/src/template/internal/loader.js";
-import { createFeaturesJson } from "./createFeaturesJson.js";
 import { Command } from "commander";
+import { createTemplatesJson } from "../templates/createTemplatesJson.js";
 
 const handler = async ({ scope }: { scope: string }): Promise<void> => {
   const projectStructure = await ProjectStructure.init({ scope });
   const templateFilepaths = getTemplateFilepaths(
     projectStructure.getTemplatePaths()
   );
-  const templateModules = await loadTemplateModules(
-    templateFilepaths,
-    true,
-    false
-  );
 
-  createFeaturesJson(templateModules, projectStructure);
+  createTemplatesJson(templateFilepaths, projectStructure, "FEATURES");
```

Comment:
Could "FEATURES" and "TEMPLATES" be an enum?

Reply:
enums are advised against in Typescript for various reasons. Union types as I have here are preferred.